### PR TITLE
Added more test-envs.

### LINKED ISSUE #3: Add Build Envs
Add a build env for Django 1.9 and `djangomaster`.

### PR DESCRIPTION
**Pull request by somexpert**
Fixes #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,20 @@ language: python
...
+    - TOX_ENV=py27-django18
+    - TOX_ENV=py34-django18
+    - TOX_ENV=py27-django19
+    - TOX_ENV=py34-django19
+    - TOX_ENV=py27-djangomaster
+    - TOX_ENV=py34-djangomaster
+matrix:
+    allow_failures:
+        - env: TOX_ENV=py34-django18
+        - env: TOX_ENV=py27-django19
```
Comment (damonkelley) [inline .travis.yml]: Did you know that this is passing?

**Comment by somexpert** [comment]
ping @ldko, @damonkelley

File: tox.ini [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+envlist = py27-django1{8,9}, flake8
+
+[flake8]
+max-line-length = 99
 
 [testenv]
-deps = Django==1.8
+deps =
+    django18: Django==1.8
+    django19: Django==1.9
```
Comment (damonkelley) [inline tox.ini]: I think we actually want `=~` here. Django doesn't do a `.0` release, so `1.9` is equivalent to `1.9.0`

Comment (damonkelley) [inline tox.ini]: https://github.com/django/django/releases

Comment (somexpert) [inline tox.ini]: Ah, I see what you mean. I'll change that here and in the other repos

**Comment by damonkelley** [comment]
:+1:

**Pull request merged**